### ISSUE TITLE
[Android] Refactor `Elevation` to use `<layer-list>`

### ISSUE THREAD

**Issue by carldebilly** [unoplatform/uno#4182]
# Android Elevation
Rewrite the Elevation feature of Uno.UI.Toolkit to draw the shadow itself (like what Sharpnado is doing) instead of relying on the platform's `.SetElevation()` feature.

## For which Platform:

- [ ] iOS
- [X] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

**Comment by jeromelaban** [comment]
@carldebilly On which scenario is this useful ? Is it to avoid the ElevatedView ?

**Comment by carldebilly** [comment]
@jeromelaban Nope. The goal is to have a better control on the result. Currently there's very few parameters to the shadow and the result if pretty different than other platforms.

**Comment by MartinZikmund** [comment]
Will be implemented as part of #3586

**Comment by MartinZikmund** [comment]
Not relevant to Skia renderer targets
